<launch>
  <node name="task2" pkg="floribot_robot" type="task2_node" >
    <rosparam file="$(find floribot_robot)/config/task2.yaml" command="load" />
  </node>
</launch>